spec/14-classes.md

```diff
@@ -58,7 +58,15 @@ object. As such, assignment of a handle does not copy the object itself.
 
 <pre>
   <i>class-declaration:</i>
-    <i>class-modifier<sub>opt</sub></i>  class  <i>name   class-base-clause<sub>opt</sub>  class-interface-clause<sub>opt</sub></i>   {   <i>trait-use-clauses<sub>opt</sub>   class-member-declarations<sub>opt</sub></i> }
+    <i>class-modifier<sub>opt</sub></i>  class  <i>name   class-base-clause<sub>opt</sub>  class-interface-clause<sub>opt</sub></i>   {   <i>class-members<sub>opt</sub></i>   }
+    
+   <i>class-members:</i>
+    <i>class-member</i>
+    <i>class-members   class-member</i>
+    
+   <i>class-member:</i>
+    <i>trait-use-clauses</i>
+    <i>class-member-declarations</i>
```

Comment:
If I understand you correctly, you suggest that classes and traits use the same set of members and then have a written Constraint in traits that prohibits the presence of constants. I'm no grammarian, but the model I've tried to follow is to do as much in the grammar without using constraints unless the grammar gets unwieldy. To be sure, the dividing line is subjective, and I have no preference as to which way to go on this issue. That said, as I think my solution works, and is clear, I won't take any further action on this.